ui/lesson/TerminalChallenge.tsx

```diff
@@ -3,9 +3,9 @@
 import { useState, useEffect } from 'react'
 import { LessonDirection } from 'types'
 import { Lesson, LessonTabs, LessonTerminal } from 'ui'
-import { useMediaQuery } from 'hooks'
 import { setUserProgress } from 'lib/user'
 import { useStatus } from 'hooks'
```

Comment:
Nitpick but here you should be able to do:
```jsx
import { useStatus, useMediaQuery } from 'hooks'
```

Reply:
Thanks for the suggestion, I have changed it